src/aws-ec2/instance-connect-endpoint.ts

```diff
@@ -45,6 +45,20 @@ export interface InstanceConnectEndpointProps {
   readonly vpc: aws_ec2.IVpc;
 }
 
+export interface InstanceConnectEndpointAttributes {
+  /**
+   * The ID of the EC2 Instance Connect Endpoint.
+   */
+  readonly instanceConnectEndpointId: string;
+
+  /**
+   * The security groups associated with the EC2 Instance Connect Endpoint.
+   *
+   * @default - no security groups
+   */
+  readonly securityGroups?: aws_ec2.ISecurityGroup[];
```

Comment:
I referred to the following and made these properties optional.

https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-ec2/lib/vpc-endpoint.ts#L930

https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-ecs/lib/cluster.ts#L801

 I'm not sure which approach is preferable; making them required would also be perfectly fine.

Reply:
I see. I have not used this resource much, but from a user perspective do you think this should be required? That will help me decide this conclusion.

Reply:
Since we usually need to configure security groups for access to the instance simultaneously (by `connections.alloFrom/To()`), I think it makes sense for the Connection class to have a SecurityGroup. I'll make it a required parameter!

Reply:
Thanks!